host_vars/fffw-lebenshilfe-core/base.yml

```diff
@@ -0,0 +1,8 @@
+---
+location: fffw-lebenshilfe
+role: corerouter
+target: "ipq40xx/generic"
+model: "avm_fritzbox-7530"
+
+wireless_profile: freifunk_fw
```

Comment:
you could consider setting the wireless_profile within your location group_vars. saving some redundant configuration.

we can also think of introducing a group per community. therefore you could override the wireless_profile for all hosts belonging to a community.

Reply:
The latter solution sounds really attactive to me. For the sake of getting the site online fast, I'd go for the first solution for now. :)

Reply:
As I had difficulties at defining the wifi profile in group-vars, I left them in host-vars for now.